tests.rb

```diff
@@ -0,0 +1,139 @@
+require_relative './article_handler.rb'
+require_relative './repository_handler.rb'
+require "minitest/autorun"
+
+describe ArticleHandler do
```

Comment:
@arielj Maybe it would be cleaner to break this test into two files. One for `ArticleHandler` and one for `RepositoryHandler`?